lib/thin/backends/base.rb

```diff
@@ -35,7 +35,7 @@ def threadpool_size=(size)
       def threaded?; @threaded end
             
       # Allow using SSL in the backend.
-      attr_writer :ssl, :ssl_options
+      attr_writer :ssl, :ssl_options, :ssl_version, :ssl_cipher_list
```

Comment:
Those two new attr don't seem to be used anywere. Version and cypher_list are stored in `ssl_options`.